README.rst

```diff
@@ -145,6 +145,54 @@ Once all the known validator callbacks have been registered, use the
 This will throw a ``ValidationException`` if the token could not be
 validated.
 
+Enforcing SciTokens Logic
+-------------------------
+For most users of SciTokens, determining that a token is valid is insufficient.
+Rather, most will be asking "does this token allow the current resource
+request?"  The valid token must be compared to some action the user is
+attempting to take.
+
+To assist in the authorization enforcement, the SciTokens library provides
+the ``Enforcer`` class.
+
+An unique Enforcer object is needed for each thread and issuer:
+
+::
+
+    >>> enf = scitokens.Enforcer("https://scitokens.org/dteam")
+
+This object will accept tokens targetted to any audience; a more typical
+use case will look like the following:
+
+::
+
+    >>> enf = scitokens.Enforcer("https://scitokens.org/dteam",
+                                 audience="https://example.com")
+
+This second enforcer would not accept tokens that are intended for
+https://google.com.
+
+The enforcer can then test authorization logic against a valid token:
+
+::
+
+    >>> token = "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtp..."
+    >>> stoken = scitokens.SciToken.deserialize(token)
+    >>> enf.generate_acls(stoken)
```

Comment:
Should we mention the `generate_acls` function separate from `test`.  The `test` function is such a logically simple concept, while `generate_acls` isn't quite.

Reply:
Yeah - my thought was to link it to the more detailed API docs ... didn't want to get into the depths of this function in the README.  Let's delay this until later?

Reply:
Sure, works for me.